test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -973,4 +973,73 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
 				"Eventually failed waiting for AWSCluster to show VPC endpoint as deleted in conditions")
 		})
 	})
+
+	ginkgo.Describe("Dedicated hosts cluster test", func() {
```

Comment:
For the time being can we disable the test
 
```suggestion
        // NOTE: disabled whilst discussions happen on the costs.
	ginkgo.PDescribe("Dedicated hosts cluster test", func() {
```

Reply:
should I just comment this test out? it is still failing.